static/css/components/buttonCta.less

```diff
@@ -21,12 +21,45 @@ a.cta-btn {
   background-color: @mid-grey;
   line-height: 1.5em;
 
+  &:link {
+    color: @white;
+    text-decoration: none;
+  }
+
+  &:visited {
+    color: @white;
+    text-decoration: none;
+  }
+
   &--available {
     background-color: @primary-blue;
+    color: @white;
   }
+
   &--unavailable {
     background-color: @orange;
+    color: @white;
+  }
+
+  &--sponsor {
+    background: @white-blue;
+    border: 2px solid @link-blue;
+  }
+
+  &--sponsor:link {
+      color: @link-blue;
   }
```

Comment:
@mekarpeles the conflict in this file 
``` 
=======
  &--preview, &--preview:link, &--preview:visited {
    background-color: @white;
    border: 2px solid @primary-blue;
    color: @primary-blue;
  }
>>>>>>> master-sponsorship
``` 
Needs to be kept. These changes were made by @shivam to fix the bug